src/components/forms/badge-form.js

```diff
@@ -236,7 +236,7 @@ class BadgeForm extends React.Component {
     const badgeType = currentSummit.badge_types.find(
       (bt) => bt.id === entity.type_id
     );
-    const access_levels = badgeType.access_levels
+    const access_levels = currentSummit.badge_access_level_types
```

Comment:
this is not correct
u need to use the badgeType.access_levels  ids and find those on currentSummit.badge_access_level_types and map accordingly